app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
+const { getCategoryObjects } = require('./controllers/categories-controller.js');
+
+app.use(express.json());
+
+//GET
+app.get('/api/categories', getCategoryObjects);
+
+//Custom Errors
+app.use((err, req, res, next) => {
+    if(err.status && err.msg){
+        res.status(err.status).send({msg: err.msg});
```

Comment:
This error handler looks good, however it's not yet relevant to the current ticket. Please remove this for now, then introduce it alongside the corresponding behavior of a related ticket.